### DISCUSSION TITLE
Api comand to remove offline files

### DISCUSSION THREAD

**Discussion by ydrob** [immich-app/immich#6187] [Q&A] [answered]
I use immich with external library, in that case it doesn't scan for new files, I do it by api command  
curl \
    --request GET \
    --url "$endpoint" \
    --header "x-api-key: $api_key" |
    jq --raw-output '.[] | select(.type=="EXTERNAL") | .id' |
    xargs -I{} curl \
        --request POST \
        -vvv \
        --url "$endpoint/{}/scan" \
        --data-raw "{}" \
        --header "x-api-key: $api_key"
 Is there some trick to do in the same way for removing offline files? This option exists in immich web, but I want to automate it.
 
![image](https://github.com/immich-app/immich/assets/77321248/7295f353-4f20-4b0f-8796-5b1d9796b4aa)

**Comment by bo0tzz** [comment, marked as answer]
https://immich.app/docs/api/remove-offline-files

**Reply by Sums123** [reply]
Hi, not sure what I'm doing wrong. I'm using Postman to send the request

`http://192.168.178.5:2283/api/library/a4ac6b76-ff79-4e1a-be0f-d3debc9697cb/removeOffline`

Bearer Token is set to the correct value and I receive a 201, however the offline files are not removed

**Comment by DanTulovsky** [comment]
It looks like https://immich.app/docs/api/remove-offline-files doesn't exist anymore.. is there an updated call?
